OSNetworkRequirementChecker-HLApp/src/app_manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "SchemaVersion": 1,
+  "Name": "DiagnosticApp",
+  "ComponentId": "8128f290-f4b1-4986-945c-1868d38e33a5",
+  "EntryPoint": "/bin/app",
+  "CmdArgs": [],
+  "Capabilities": {
+    "AllowedConnections": [ 
+      "eastus-prod-azuresphere.azure-devices.net",
```

Comment:
Can you add these endpoints from https://docs.microsoft.com/en-us/azure-sphere/network/ports-protocols-domains?
global.azure-devices-provisioning.net
global.azure-devices-provisioning.net

Also need to add these new ones?
            "anse.azurewatson.microsoft.com",
            "sphereblobeus.azurewatson.microsoft.com",
            "sphereblobweus.azurewatson.microsoft.com"

Reply:
updated.